### PR TITLE
Add support for tracking playtime statistics

### PR DESCRIPTION
**Pull request by gablm**
- Adds tracking of various statistics related to playtime: Time spent daily, weekly and monthly playing a certain game plus the length of the most recent session.

- Moves playtime related code from the HomePage to it's own class, which can be accessed via the regions GamePresetProperty.

### CODE REVIEW

File: CollapseLauncher/XAMLs/MainApp/Pages/HomePage.xaml.cs [hunk 1]
```diff
@@ -1019,12 +1022,12 @@ private async void CheckRunningGameInstance(CancellationToken Token)
                             IGameSettingsUniversal gameSettings = CurrentGameProperty!._GameSettings!.AsIGameSettingsUniversal();
                             PresetConfig gamePreset = CurrentGameProperty._GamePreset;
 
-                            StartPlaytimeCounter(currentGameProcess, gamePreset, fromActivityOffset);
+                            CurrentGameProperty!._GamePlaytime!.StartSession(currentGameProcess);
```
Comment (bagusnl) [inline CollapseLauncher/XAMLs/MainApp/Pages/HomePage.xaml.cs:1025]: I think we can move the StartSession call into CheckRunningGameInstance so the playtime can be tracked if the game is started before Collapse does

**Review by bagusnl — COMMENTED** [review]
Overall all the codes are fine by me, just some suggestion I added in the comments.

If you can please fix the Qodana warnings
Good job on the work!

File: CollapseLauncher/XAMLs/MainApp/Pages/HomePage.xaml.cs (hunk 1 above)
Comment (gablm) [reply CollapseLauncher/XAMLs/MainApp/Pages/HomePage.xaml.cs:1025]: isn't that call already inside CheckRunningGameInstance?

Comment (bagusnl) [reply CollapseLauncher/XAMLs/MainApp/Pages/HomePage.xaml.cs:1025]: Oh ye.... My bad didn't see it xd
All good then. We'll wait for neon's check and all good

**Review by bagusnl — APPROVED** [review]
All good on me~

**Review by neon-nyan — APPROVED** [review]
I abbrob
image

**Pull request merged**